### PR TITLE
Adding documentation for the operators: lod_tensor_to_array , array_to_lod_tensor, create_array, increment

### PR DESCRIPTION
**Pull request by kavyasrinet**
This PR has added documentation for the following:
1.  `lod_tensor_to_array`
2.  `array_to_lod_tensor`
3.  `create_array`
4.  `increment`

### CODE REVIEW

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -440,9 +440,23 @@ def topk(input, k):
...
+        table (LoDRankTable): The RankTable that provides the coarse lod
+                              infomation to build the output LoDTensorArray.
+
+    Returns:
+        Variable: The tensor variable of type LOD_TENSOR.
+
+    Examples:
+        .. code-block:: python
+
+        x = fluid.layers.data(name='x', shape=[10])
```
Comment (sidgoyal78) [inline python/paddle/v2/fluid/layers/control_flow.py]: I think for statements in Line457-459, there should be another level of indentation (1 level deeper than the ".. code-block:: python) for proper rendering. Can you check?

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -458,9 +472,24 @@ def lod_tensor_to_array(x, table):
...
+    Returns:
+        Variable: The array variable of type LOD_TENSOR_ARRAY.
+
+    Examples:
+        .. code-block:: python
+
+        x = fluid.layers.data(name='x', shape=[10])
+        table = fluid.layers.lod_rank_table(x, level=0)
+        array = fluid.layers.lod_tensor_to_array(x, table)
+        lod_tensor = fluid.layers.array_to_lod_tensor(array, table)
```
Comment (sidgoyal78) [inline python/paddle/v2/fluid/layers/control_flow.py]: Same as above.

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -473,10 +502,24 @@ def array_to_lod_tensor(x, table):
...
+
+    Returns:
+        Variable: The tensor variable storing the transformation of
+        element-wise increment of each value in the input.
+
+    Examples:
+        .. code-block:: python
+
+        data = fluid.layers.data(name='data', shape=[32, 32], dtype='float32')
+        data = fluid.layers.increment(x=data, value=3.0, in_place=True)
```
Comment (sidgoyal78) [inline python/paddle/v2/fluid/layers/control_flow.py]: same as above.

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 4]
```diff
@@ -511,6 +554,21 @@ def array_write(x, i, array=None):
...
+    Args:
+        dtype (data type): The data type of the elements in the array.
+
+    Returns:
+        Variable: The tensor variable storing the elements of data type.
+
+    Examples:
+        .. code-block:: python
+
+        data = fluid.layers.create_array(dtype='float32')
```
Comment (sidgoyal78) [inline python/paddle/v2/fluid/layers/control_flow.py]: same as above.

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 1 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/layers/control_flow.py]: Yeah it was a formatting issue. Thanks for noticing this!

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 2 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/layers/control_flow.py]: Fixed

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 3 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/layers/control_flow.py]: Fixed.

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 4 above)
Comment (kavyasrinet) [reply python/paddle/v2/fluid/layers/control_flow.py]: Fixed.

**Pull request merged**